src/proposals/api/views.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import views
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+from core.authentication import TokenAuthentication
+from proposals.models import TalkProposal, TutorialProposal
+from reviews.models import Review
+
+class ProposalAPIView(views.APIView):
+
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        proposed_talk_num = TalkProposal.objects.count()
+        proposed_tutorial_num = TutorialProposal.objects.count()
```

Comment:
有個欄位 `cancelled` 代表 submitter 投稿後有沒有收回，應該要濾掉被收回的 proposal
ref: https://github.com/pycontw/pycon.tw/blob/master/src/proposals/models.py#L152-L156
```suggestion
        proposed_talk_num = TalkProposal.objects.filter(cancelled=False).count()
        proposed_tutorial_num = TutorialProposal.objects.filter(cancelled=False).count()
```